code/iaas/auth-logic/src/main/java/io/cattle/platform/iaas/api/auth/integration/github/GithubConfigManager.java

```diff
@@ -92,6 +100,19 @@ public GithubConfig getCurrentConfig(Map<String, Object> config) {
         if (config.get(CLIENT_ID) != null) {
             clientId = (String) config.get(CLIENT_ID);
         }
+        if (((Policy) ApiContext.getContext().getPolicy()).isOption(Policy.AUTHORIZED_FOR_ALL_ACCOUNTS)) {
+            if (StringUtils.isBlank(
+                    (String) ApiContext.getContext().getApiRequest().getAttribute(GithubConstants.GITHUB_ACCESS_TOKEN))) {
+                Identity gotIdentity = Identity.fromId(ArchaiusUtil.getString(SecurityConstants.AUTH_ENABLER).get());
```

Comment:
The reason that this as been added is so that if there is not an Apikey avaliable for github for a user who is an admin and is listing github config we are able to access github using the person who turned github auth on. That way there are no 500's due to NPE. We could also just not list the saved identities, however this does not seem like a good options/ the right one.